cmd/goal/clerk.go

```diff
@@ -69,10 +69,10 @@ var (
 	requestFilename    string
 	requestOutFilename string
 
-	simulateAllowEmptySignatures   bool
-	simulateAllowMoreLogging       bool
-	simulateAllowExtraOpcodeBudget bool
-	simulateExtraOpcodeBudget      uint64
+	simulateAllowEmptySignatures  bool
+	simulateAllowMoreLogging      bool
+	simulateAllowMoreOpcodeBudget bool
+	simulateExtraOpcodeBudget     uint64
```

Comment:
didn't want to also change this one?

Reply:
I want to slightly push back on changing this one, for I want the boolean flags (for convenience of extra budget) to align with `--allow-more-logging`, but for specifying a particular budget, we stay the same.

Reply:
ok, makes sense.